src/ast-writer.cc

```diff
@@ -69,6 +71,8 @@ struct Context {
   int func_type_index;
 };
 
+}  // namespace
```

Comment:
Using the anonymous namespace is standard practice in C++ for static variables, since all variables in the anonymous namespace is automatically static.

However, I forgot to notice you should remove the "static" because the name is in the anonymous namespace (again, standard C++ practice).